diff of execution of:
echo 'cat <<EOF; $PWD and $OLDPWD;ha;$PATH ;EOF' | tr ';' '\n' | [shell]
--- expected	2024-08-27 17:55:03.744547587 +0200
+++ found	2024-08-27 17:55:03.770547980 +0200
@@ -1,8 +1,11 @@
- /home/bruno/Github42/Minishell/test/temp and /home/bruno/Github42/Minishell/test
+ $PWD and $OLDPWD
 ha
-/home/bruno/.local/bin:/home/bruno/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin 
+$PATH 
 
 ======== stderr =======
-
+[progname]:  Is a directory
+[progname]: ha: command not found
+[progname]: /home/bruno/.local/bin:/home/bruno/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin:/home/bruno/.local/bin: No such file or directory
+[progname]: EOF: command not found
 ======== return =======
 return value: 0
